pkg/upgrade/consts.go

```diff
@@ -31,6 +31,9 @@ const (
 	UpgradeWaitForPodCompletionStartTimeAnnotationKeyFmt = "nvidia.com/%s-driver-upgrade-wait-for-pod-completion-start-time"
 	// UpgradeValidationStartTimeAnnotationKeyFmt is the format of the node annotation indicating start time for validation-required state
 	UpgradeValidationStartTimeAnnotationKeyFmt = "nvidia.com/%s-driver-upgrade-validation-start-time"
+	// UpgradeRequestedLabelKeyFmt is the format of the node label key indicating driver upgrade was requested (used for orphaned pods)
+	// Setting this label will trigger setting upgrade state to upgrade-required
+	UpgradeRequestedLabelKeyFmt = "nvidia.com/%s-driver-upgrade-requested"
```

Comment:
Isn't this an annotation? Shouldn't it be named `UpgradeRequestedAnnotationKeyFmt`?

Reply:
Done